temporian/core/operators/arithmetic/__init__.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=unused-import
+
+from temporian.core.operators.arithmetic.base import Resolution
+from temporian.core.operators.arithmetic.addition import add, AdditionOperator
+from temporian.core.operators.arithmetic.division import (
+    divide,
+    DivisionOperator,
+)
+from temporian.core.operators.arithmetic.subtraction import (
+    subtract,
+    SubtractionOperator,
+)
+from temporian.core.operators.arithmetic.multiplication import (
+    multiply,
+    MultiplicationOperator,
+)
```

Comment:
I'd rename these operators to the simple present form of the verb, rather than the noun (e.g. `DivisionOperator` -> `DivideOperator`) to keep it consistent with existing ones (we have `PropagateOperator`, `SelectOperator`, `GlueOperator`, etc. rather `PropagationOperator`, `SelectionOperator`, etc.)

Reply:
same for the naming of the numpy implementations

Reply:
Yep, I agree, will do.